docs/topics/rn-resolved-issues.adoc

```diff
@@ -7,36 +7,20 @@
 
 At the time of the release the following resolved issues have been identified as major issues worth highlighting.
 
-For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12360214[{ProductVersion} resolved issues] in Jira.
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12364384[{ProductVersion} resolved issues] in Jira.
 
 .Major resolved issues
 [cols="25%,75%",options="header"]
 |====
 |ID
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUP-2941[WINDUP-2941]
-|MTA reports load slowly, do not include Target Runtime labels on the *Application List* screen, and do not have working feedback controls.
+|link:https://issues.redhat.com/browse/WINDUP-3082[WINDUP-3082]
+|CodeReady Studio and Eclipse IDEs: A `NullPointerException` is thrown when browsing rules in *Rulesets* view.
```

Comment:
Change to: `NullPointerException` error occurs when browsing rules in the Rulesets view.